src/main/java/wooteco/subway/admin/service/LineService.java

```diff
@@ -21,34 +30,106 @@ public LineService(LineRepository lineRepository, StationRepository stationRepos
         this.stationRepository = stationRepository;
     }
 
-    public Line save(Line line) {
-        return lineRepository.save(line);
+    public LineResponse save(LineRequest lineRequest) {
+        try {
+            return LineResponse.of(lineRepository.save(lineRequest.toLine()));
+        } catch (DbActionExecutionException e) {
+            throwDuplicateKeyException(e);
+            throw new IllegalArgumentException(e.getMessage());
+        }
     }
 
-    public List<Line> showLines() {
-        return lineRepository.findAll();
+    private void throwDuplicateKeyException(DbActionExecutionException e) {
```

Comment:
```suggestion
    private void checkDuplicateKeyException(DbActionExecutionException e) {
```

이름으로 봤을 땐 무조건 throw할 것 같아요!